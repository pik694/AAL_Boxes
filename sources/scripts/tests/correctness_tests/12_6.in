12
59 18 42
75 94 51
55 58 66
100 98 28
58 68 8
8 89 20
91 100 4
61 35 3
33 73 59
13 20 11
39 4 61
9 86 51

