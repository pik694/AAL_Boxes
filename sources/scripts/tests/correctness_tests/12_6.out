9 0 2 7 4 8 1 10 11 3 5 6 
1171672->895130
